docs/next.config.js

```diff
@@ -43,6 +43,14 @@ module.exports = withDocsInfra({
     return {
       ...config,
       plugins,
+      resolve: {
```

Comment:
See https://github.com/mui/material-ui/pull/32107#issuecomment-1180889892
I don't really understand why this change is necessary